### PR TITLE
Droth 1598 tn its stop sign

### PR DESCRIPTION
**Pull request by joanacoelho**

### CODE REVIEW

**Comment by joanacoelho** [comment]
ok

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PointAssetOperations.scala [hunk 1]
```diff
@@ -84,17 +106,38 @@ trait PointAssetOperations {
...
   def update(id:Long, updatedAsset: IncomingAsset, geometry: Seq[Point], municipality: Int, username: String, linkSource: LinkGeomSource): Long
   def setAssetPosition(asset: IncomingAsset, geometry: Seq[Point], mValue: Double): IncomingAsset
   def toIncomingAsset(asset: IncomePointAsset, link: RoadLink) : Option[IncomingAsset] = { throw new UnsupportedOperationException()}
-
   def getByBoundingBox(user: User, bounds: BoundingRectangle): Seq[PersistedAsset] = {
     val roadLinks: Seq[RoadLink] = roadLinkService.getRoadLinksWithComplementaryFromVVH(bounds)
     getByBoundingBox(user, bounds, roadLinks, Seq(), floatingTreatment)
   }
 
+  def assetProperties(pointAsset: PersistedPointAsset, since: DateTime) : Map[String, Any] = { throw new UnsupportedOperationException("Not Supported Method") }
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PointAssetOperations.scala]: this method probably could be deleted, not used

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/TrafficSignService.scala [hunk 2]
```diff
@@ -238,6 +242,28 @@ class TrafficSignService(val roadLinkService: RoadLinkService, val userProvider:
     }
   }
 
+  override def getChanged(sinceDate: DateTime, untilDate: DateTime): Seq[ChangedPointAsset] = { throw new UnsupportedOperationException("Not Supported Method, Try to used") }
```
Comment (MAC-CGI) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/TrafficSignService.scala]: commented on previous pull request (warning signs)

File: digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PointAssetOperations.scala [hunk 3]
```diff
@@ -66,6 +70,24 @@ trait PersistedPointAsset extends PointAsset with IncomingPointAsset {
   val linkSource: LinkGeomSource
 }
 
+trait PersistedPoint extends PersistedPointAsset with IncomingPointAsset {
```
Comment (oskarmalmstrom) [inline digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PointAssetOperations.scala]: with IncomingPointAsset is needed here?

**Comment by oskarmalmstrom** [comment]
Following branches were merged into one branch TNITS_deploy and deployed. Closing this pull request without merging.  
- DROTH-486
- DROTH-1575
- DROTH-507
- DROTH-1599
- DROTH-1598
- DROTH-1612

**Pull request closed without merging**